farm/train.py

```diff
@@ -464,7 +467,7 @@ def _save(self):
             pickle_module=dill,
         )
 
-        checkpoint_name = f"epoch_{self.from_epoch}_step_{self.from_step}"
+        checkpoint_name = f"epoch_{self.from_epoch + 1}_step_{self.from_step}"
```

Comment:
This is only to make the checkpoint folder names more readable. It doesn't affect the loading/saving behaviour of checkpoints.

Reply:
Hm... so we store a checkpoint as "epoch 1" but need to load it via from_epoch=0? This sounds confusing to me.